src/math/fibonacci.lua

```diff
@@ -0,0 +1,80 @@
+-- Fibonacci.lua
+
+-- Fibonacci Sequence:
+-- The Fibonacci sequence is a series of numbers where each number (after the first two) is the sum of the two preceding ones.
+-- More information: https://en.wikipedia.org/wiki/Fibonacci_number
+
+-- Author: Gyandeep
```

Comment:
Not useful to readers of the code. Your name will be in the commit history if/when this is merged.